ci/test/00_setup_env_native_tsan.sh

```diff
@@ -7,7 +7,6 @@
 export LC_ALL=C.UTF-8
 
 export CONTAINER_NAME=ci_native_tsan
-export DOCKER_NAME_TAG=ubuntu:16.04
```

Comment:
Please update the travis yaml as well

Reply:
Also the workaround in line 18 can be removed

Reply:
Also release notes are needed to say that xenial is no longer supported with system libs

Reply:
> Also release notes are needed to say that xenial is no longer supported with system libs

In which way: wiki or pr?

Reply:
Building and running Bitcoin Core on xenial works fine today on master and 0.20.0. Your pull is changing that.

Reply:
I mean: in which way add this info to release notes?  :)

Reply:
I'd speak against backporting this to 0.20.0, so it should go into the release notes for 0.21.0